src/wallet/rpcwallet.cpp

```diff
@@ -3046,6 +3049,16 @@ static RPCHelpMan listunspent()
         entry.pushKV("scriptPubKey", HexStr(scriptPubKey));
         entry.pushKV("amount", ValueFromAmount(out.tx->tx->vout[out.i].nValue));
         entry.pushKV("confirmations", out.nDepth);
+        if (!out.nDepth) {
+            size_t ancestor_count, descendant_count, ancestor_size;
+            CAmount ancestor_fees;
+            pwallet->chain().getTransactionAncestry(out.tx->GetHash(), ancestor_count, descendant_count, &ancestor_size, &ancestor_fees);
+            if (ancestor_count) {
+                entry.pushKV("ancestorcount", (uint64_t)ancestor_count);
+                entry.pushKV("ancestorsize", (uint64_t)ancestor_size);
+                entry.pushKV("ancestorfees", (uint64_t)ancestor_fees);
```

Comment:
In 47fdd185f1dca3f0a92b1b67c4523053ed2fc1a3 " RPC: Add ancestor{count,size,fees} to listunspent output"

This will output the fees in satoshis which is highly unintuitive, especially since the documentation does not say so. Everywhere else we have amounts in the RPCs, they are formatted for output with `ValueFromAmount` which converts them to the correct decimal form for output.

Reply:
Not everywhere, no, and the only other location of ancestorfees specifically also uses satoshis.

Reply:
The docs should at least indicate the units then. In most places the term "fee" is used, the expected unit is BTC, not satoshis.